src/test/run-pass/dynamic-drop.rs

```diff
@@ -111,6 +111,20 @@ fn assignment1(a: &Allocator, c0: bool) {
     _v = _w;
 }
 
+#[allow(unions_with_drop_fields)]
+union Boxy<T> {
+    a: T,
+    b: T,
+}
+
+fn union1(a: &Allocator) {
+    unsafe {
+        let mut u = Boxy { a: a.alloc() };
+        u.b = a.alloc();
+        drop(u.a);
```

Comment:
> According to the union RFC, the assignment above will drop u.b

Oh I see, it must be an addition not present in the original RFC, hence the confusion.